### DISCUSSION TITLE
How to set a Data Link for a field in the Log Panel instead of in Explore？

### DISCUSSION THREAD

**Discussion by Wudadada** [grafana/clickhouse-datasource#1005] [Q&A]
![WechatIMG1487](https://github.com/user-attachments/assets/a476d166-2b6b-4402-aa65-46a37aff19da)
like this

**Comment by Wudadada** [comment]
i have tried but it does not work
![WX20240929-091706@2x](https://github.com/user-attachments/assets/dfe37820-1b00-4491-b0a1-9cb6adac3dde)
![WX20240929-091826@2x](https://github.com/user-attachments/assets/d7884e29-0754-472d-b80e-4b6db0425b52)

**Reply by JMarques007** [reply]
Change the name of field to another name, I think depends of name in Data Source

**Comment by JMarques007** [comment]
One question how did you put the filtering feature?
![image](https://github.com/user-attachments/assets/228ea863-d3b0-460e-923b-6e95417ccb69)

**Reply by Wudadada** [reply]
> One question how did you put the filtering feature? ![image](https://private-user-images.githubusercontent.com/4114837/375731454-228ea863-d3b0-460e-923b-6e95417ccb69.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg2NTIzMjYsIm5iZiI6MTcyODY1MjAyNiwicGF0aCI6Ii80MTE0ODM3LzM3NTczMTQ1NC0yMjhlYTg2My1kM2IwLTQ2MGUtOTIzYi02ZTk1NDE3Y2NiNjkucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAxMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMTFUMTMwNzA2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NmRkOGQ3MDg2ZDVmYjU2NWY3ODExOWMyNzgzM2U3Y2M2NGU2NDMwNTZjMDViOTYyZmUwODE1OWRiZTNlNGRiNyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.iCXI66ZfJFI4pXvy9f8VUOvS0AvtcCX8-vW4cPxO0xU)

I think it may be the latest grafana or clikhouse-datasource releases version's log panel's function，I didn't configure for it but it just appeared.
